k8s/prod/common/ccpay/ccpay-payment-app.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: ccpay-payment-app
+  namespace: fees-pay
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ccpay-payment-api
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: payment-api
+    version: 0.1.0
+  values:
+    rollback:
+      enable: true
+    java:
+      replicas: 2
+      image: hmctspublic.azurecr.io/payment/api:prod-a4872e81
+      ingressHost: payment-api-prod.service.core-compute-aat.internal
+      applicationInsightsInstrumentKey: "52125c3c-ea7b-4954-a872-fe6987780a88"
+      environment:
+        SPRING_DATASOURCE_URL: "jdbc:postgresql://payment-postgres-db-prod.postgres.database.azure.com:5432/payment"
+        SPRING_DATASOURCE_USERNAME: payment@payment-postgres-db-prod
+        SPRING_LIQUIBASE_ENABLED: 'false'
+        RUN_DB_MIGRATION_ON_STARTUP: 'false'
+        IDAM_URL: https://idam-api.platform.hmcts.net
+        S2S_URL: http://rpe-service-auth-provider-prod.service.core-compute-prod.internal
+        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
+          -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -javaagent:/opt/app/applicationinsights-agent-2.3.1.jar
+      aadIdentityName: ccpay
+      keyVaults:
+        "ccpay":
+          resourceGroup: ccpay
```

Comment:
won't need it if you are on latest java chart and also, it would be good to move this to values.yaml and not override everywhere.